.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might I recommend putting `.project` in your global gitignore on your personal machine?  See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore  It helps avoid these kinds of things getting committed.